inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/pulsar/PulsarHandler.java

```diff
@@ -282,8 +282,6 @@ private CompressionType getPulsarCompressionType() {
         switch (type) {
             case "LZ4":
                 return CompressionType.LZ4;
-            case "NONE":
```

Comment:
I guess it's better to move the `case "NONE" line above the `default` line.

Reply:
replicated for the default definition, you could check the issue.

Reply:
I mean change it to this:

```java
            case "NONE":
            default:
                return CompressionType.NONE;
```

The advantage is that it can be clearly pointed out that we also support the `"NONE"` configuration item.